flycheck.el

```diff
@@ -3610,6 +3650,55 @@ list."
       ;; highlighting
       (flycheck-error-list-highlight-errors preserve-pos))))
 
+(defvar flycheck-error-list-current-filter nil
```

Comment:
I'm not sure about this variable.  Do we need an extra one in addition to `flycheck-error-list-minimum-level`?  Is there a reason why you don't just set the buffer local variable of `flycheck-error-list-minimum-level`?